doc/classes/TextureLayered.xml

```diff
@@ -81,23 +88,30 @@
 			<argument index="1" name="layer" type="int">
 			</argument>
 			<description>
+				Sets the data for specified layer. Data takes the form of a 2-dimensional [Image] resource.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="data" type="Dictionary" setter="_set_data" getter="_get_data" default="{&quot;depth&quot;: 0,&quot;flags&quot;: 4,&quot;format&quot;: 37,&quot;height&quot;: 0,&quot;layers&quot;: [  ],&quot;width&quot;: 0}">
+			Returns a dictionary with all the data used by this texture.
 		</member>
 		<member name="flags" type="int" setter="set_flags" getter="get_flags" default="4">
+			Specifies with [enum Flags] apply to this texture.
 		</member>
 	</members>
 	<constants>
 		<constant name="FLAG_MIPMAPS" value="1" enum="Flags">
+			Texture will generate mip-maps on creation.
```

Comment:
We seem to be writing it as "mipmap" everywhere else, so this should be changed for consistency:

```suggestion
			Texture will generate mipmaps on creation.
```

Reply:
Thanks!